crates/turborepo-lib/src/snapshots/turborepo_lib__opts__test__turbo_boundaries_--filter_foo.snap

```diff
@@ -0,0 +1,72 @@
+---
+source: crates/turborepo-lib/src/opts.rs
+expression: opts
+---
+{
+  "repo_opts": {
+    "root_turbo_json_path": "turbo.json",
+    "allow_no_package_manager": false,
+    "allow_no_turbo_json": false
+  },
+  "api_client_opts": {
+    "api_url": "https://vercel.com/api",
+    "timeout": 30,
+    "upload_timeout": 60,
+    "token": null,
+    "team_id": null,
+    "team_slug": null,
+    "login_url": "https://vercel.com",
+    "preflight": false
+  },
+  "cache_opts": {
+    "cache_dir": ".turbo/cache",
+    "cache": {
+      "local": {
+        "read": true,
+        "write": true
+      },
+      "remote": {
+        "read": false,
+        "write": false
+      }
+    },
+    "workers": 10,
+    "remote_cache_opts": {
+      "unused_team_id": null,
+      "signature": false
+    }
+  },
+  "run_opts": {
+    "tasks": [],
+    "concurrency": 10,
+    "parallel": false,
+    "env_mode": "strict",
+    "cache_dir": ".turbo/cache",
+    "framework_inference": false,
+    "profile": null,
+    "continue_on_error": false,
+    "pass_through_args": [],
+    "only": false,
+    "dry_run": null,
+    "graph": null,
+    "daemon": null,
+    "single_package": false,
+    "log_prefix": "Task",
+    "log_order": "Stream",
+    "summarize": false,
+    "experimental_space_id": null,
+    "is_github_actions": false,
```

Comment:
I think you want to reduce the data caught in these snapshots. `is_github_actions` and `log_order` both get affected depending on where the tests will get run.